conda/recipes/libcuvs/meta.yaml

```diff
@@ -248,6 +248,9 @@ outputs:
         - {{ compiler('cuda') }}
         - cuda-cudart-dev
         - libcublas-dev
+        - libcurand-dev
```

Comment:
These changes align `libcuvs-examples` with the `-tests` and other packages above. These are needed to build the examples, now that we're removing `libraft`. Previously `libraft` was supplying the CUDA math library dependencies.